test/tests/example.js

```diff
@@ -1,46 +1,52 @@
-describe("Simple tests examples", function() {
-  it("should detect true", function() {
+describe("Simple tests examples", function () {
```

Comment:
Noticed a style deviation here. Converted from `function() {` to `function () {`. Unfortunately no style/lint rule caught this, sorry about that.